tests/test_verify.py

```diff
@@ -8,7 +8,7 @@
 
 logger = Logger()
 
-models = ["VGG-Face", "Facenet", "Facenet512", "ArcFace", "GhostFaceNet"]
+models = ["VGG-Face", "Facenet", "Facenet512", "ArcFace", "GhostFaceNet", "Facenet512ONNX"]
```

Comment:
Seems you added an optional facial recognition model. If you add it here, it will get an exception in the pipeline. So, optional models should not be tested.

Reply:
As i mentioned, pipeline failed - https://github.com/serengil/deepface/actions/runs/10579001173/job/29310896976?pr=1322

Reply:
okay i have removed it, but tested on my end